src/offline.h

```diff
@@ -48,6 +48,26 @@ class PACKAGEKITQT_LIBRARY Offline : public QObject
     };
     Q_ENUM(Action)
 
+    /**
+     * Wrapper class representing the returning value of getResults()
+     */
+    class Results
+    {
+        friend class Offline;
+    public:
+        Results(const QDBusPendingCall &call);
+
+        bool isError() const;
+        bool success() const;
+        QStringList packageIds() const;
+        Transaction::Role role() const;
+        qulonglong timeFinished() const;
+        Transaction::Error error() const;
+        QString errorDescription() const;
+    private:
+        QDBusPendingReply<bool, QStringList, unsigned, qulonglong, unsigned, QString> m_reply;
```

Comment:
Why can't we use the enum value like we used to? Was it never tested to work?

Use `quint32` instead of just `unsigned` which isn't super well specified.